lib/run.js

```diff
@@ -32,10 +32,14 @@ function validateOpts(options) {
     
     validateNumberProp(options, 'rate');
     validateNumberProp(options, 'concurrent');
-    
-    // hack to enable concurrency mode,
-    // since forkmaster expects this to be defined
-    options.rate = options.rate || 0;
+
+    if (options.concurrent) {
+        delete options.rate;
+    }
+
+    if (options.rate < 0 || Math.floor(options.rate * (options.duration / 1000)) < 1) {
+        throw new Error('duration is too low');
```

Comment:
This is really the fault of my weird fraction system to figure out how many tests to run when. When the fraction converges to 0, that doesn't necessarily mean I should run 0 tests.

Most similar systems, I think, actually start a test and then attempt to figure out if/when they should start another one. I was just too lazy to read Go code to figure it out.

Am I crazy?